test/PrintAsObjC/Inputs/Foundation.framework/Headers/Foundation.h

```diff
@@ -0,0 +1 @@
+#include <objc/NSObject.h>
```

Comment:
Hm, it's kind of weird to be putting this in PrintAsObjC/Inputs/ if tests from other folders are going to be using it too. Maybe it'd be better to put it in the normal "mock SDK" (test/Inputs/clang-importer-sdk/Frameworks/ or something) and have it include `<Foundation.h>` there?

Reply:
Thank you for reviewing it. Yeah that makes sense, it felt weird to me also to point to this file from other tests. I'll move the file and and have it include `<Foundation.h>` instead.